XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -132,5 +135,18 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <Compile Include="**\*.tizen.cs" />
     <Compile Include="**\*.tizen.*.cs" />
+    <Compile Remove="**\Snackbar.shared.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('net472')) ">
+    <Compile Include="**\*.wpf.cs" />
+    <Compile Include="**\*.wpf.*.cs" />
+    <Compile Remove="**\Snackbar.shared.cs" />
```

Comment:
do we need this line?

Reply:
We need to remove shared file from all platforms

Reply:
We need to remove shared file from all platforms